app.py

```diff
@@ -0,0 +1,383 @@
+import streamlit as st
+import pickle
+import os
+from langchain.embeddings import HuggingFaceEmbeddings
+from langchain.vectorstores import FAISS
+from langchain_google_genai import ChatGoogleGenerativeAI
+
+from shirabasusearch import search,or_search,or_list_search,word_search,hyoka_search,ids_union 
+path1 = 'data2\shirabasu_classtype.pkl'
+with open(path1, 'rb') as f:
+    classtypes = pickle.load(f)
+path2 = 'data2\shirabasu_departments.pkl'
+with open(path2, 'rb') as f:
+    departments = pickle.load(f)
+path3 = 'data2\shirabasu_fulltexts.pkl'
+with open(path3, 'rb') as f:
+    fulltexts = pickle.load(f)
+path4 = 'data2\shirabasu_hyoka.pkl'
+with open(path4, 'rb') as f:
+    hyoka = pickle.load(f)
+path5 = 'data2\shirabasu_keywordtexts.pkl'
+with open(path5, 'rb') as f:
+    keywordtexts = pickle.load(f)
+path6 = 'data2\shirabasu_metadatas.pkl'
+with open(path6, 'rb') as f:
+    metadatas = pickle.load(f)
+path7 = 'data2\shirabasu_texts.pkl'
+with open(path7, 'rb') as f:
+    texts = pickle.load(f)
+path8 = 'data2\shirabasu_yojigen.pkl'
+with open(path8, 'rb') as f:
+    yojigen = pickle.load(f)

```

Comment:
バックスラッシュではなくスラッシュにしてください
```suggestion
path1 = 'data2/shirabasu_classtype.pkl'
with open(path1, 'rb') as f:
    classtypes = pickle.load(f)
path2 = 'data2/shirabasu_departments.pkl'
with open(path2, 'rb') as f:
    departments = pickle.load(f)
path3 = 'data2/shirabasu_fulltexts.pkl'
with open(path3, 'rb') as f:
    fulltexts = pickle.load(f)
path4 = 'data2/shirabasu_hyoka.pkl'
with open(path4, 'rb') as f:
    hyoka = pickle.load(f)
path5 = 'data2/shirabasu_keywordtexts.pkl'
with open(path5, 'rb') as f:
    keywordtexts = pickle.load(f)
path6 = 'data2/shirabasu_metadatas.pkl'
with open(path6, 'rb') as f:
    metadatas = pickle.load(f)
path7 = 'data2/shirabasu_texts.pkl'
with open(path7, 'rb') as f:
    texts = pickle.load(f)
path8 = 'data2/shirabasu_yojigen.pkl'
with open(path8, 'rb') as f:
    yojigen = pickle.load(f)

```